server/routes/basicDetails.ts

```diff
@@ -33,7 +33,31 @@ export default function basicDetailsRoutes(
     const ndeliusIntegrationApiClient = new NdeliusIntegrationApiClient(token)
 
     const breachNotice = await breachNoticeApiClient.getBreachNoticeById(req.params.id as string)
-    const basicDetails = await ndeliusIntegrationApiClient.getBasicDetails(breachNotice.crn, req.user.username)
+    let basicDetails: BasicDetails = null
+
+    try {
+      basicDetails = await ndeliusIntegrationApiClient.getBasicDetails(breachNotice.crn, breachNotice.id)
+    } catch (error) {
+      if (error.status === 400 && error.data.message.includes('No home area found')) {
```

Comment:
Might be worth adding a null-check here, in case you get a different error response that doesn't contain the message attribute - you can just add question marks:
```suggestion
      if (error.status === 400 && error.data?.message?.includes('No home area found')) {
```

Reply:
null check added